build.sh

```diff
@@ -23,6 +40,31 @@ docker image build ./$IMAGE_DIR --tag $REPO/$IMAGE_DIR:$TAG_NAME --tag $REPO/$IM
     && docker push $REPO/$IMAGE_DIR:$TAG_NAME \
     && docker push $REPO/$IMAGE_DIR:$VERSION
 
-docker rm $REPO/$IMAGE_DIR
+#update remote documentation
+#will fail if you are not gcloud authed as dspci-wb-gcr-service-account
+    #docker run --rm  -v ~/.vault-token:/root/.vault-token:ro broadinstitute/dsde-toolbox:latest vault read --format=json secret/dsde/dsp-techops/common/dspci-wb-gcr-service-account.json | jq .data > dspci-wb-gcr-service-account.json
+    #gcloud auth activate-service-account --key-file=dspci-wb-gcr-service-account.json
```

Comment:
are these meant to stay commented out?

Reply:
I updated this in another PR but I think this got applied as a change over the rebase. Thanks